packages/@contentful--app-scripts/lib/create-app-definition/create-app-definition.js

```diff
@@ -36,18 +36,24 @@ function assertValidArguments(accessToken, appDefinitionSettings) {
   if (!isPlainObject(appDefinitionSettings) || !has(appDefinitionSettings, 'locations')) {
     throwValidationException(
       'AppDefinitionSettings',
-      `Expected plain object with 'location' property, got ${JSON.stringify(appDefinitionSettings, null, 2)}`
-        `Example: ${JSON.stringify({
+      `Expected plain object with 'location' property, got ${JSON.stringify(
+        appDefinitionSettings,
+        null,
+        2
+      )}``Example: ${JSON.stringify(
```

Comment:
Does this syntax with double ` work?

Reply:
I actually haven't changed something here it looks like it came from prettier when I changed something else in the file, so it probably will work when it worked before :D

Reply:
Well it was a double ` before, so it didn't change. But e.g. `console.log(`test``test`)` throws an error for me as that's the syntax for template literals: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals

Reply:
Yes you are right, I think there was a `,` lost as `throwValidationException` is also accepts three arguments. 
I added it now that the example is the `details` in the function